pkg/plugin/version_test.go

```diff
@@ -0,0 +1,302 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import (
+	"sort"
+	"testing"
+
+	g "github.com/onsi/ginkgo" // An alias is required because Context is defined elsewhere in this package.
```

Comment:
we do not use an alias for this import across the project. I'd prefer we keep all whiteout alias in this scenario.

Reply:
In `pkg/plugin/interfaces.go` we are defining a `Context` type. Therefore we can not use a `_` as the alias because `ginkgo.Context` would conflict with it.

P.S.: I did not add this change, it was part of the previous code, I just moved it.